### ISSUE TITLE
Train New Model

### ISSUE THREAD

**Issue by amirzenoozi** [lakshaychhabra/NSFW-Detection-DL#2]
Hi, Thanks For Your Clean Code
Do You Have Any Python Script That Let Me Train The New Model Based on a new dataset?

**Comment by lakshaychhabra** [comment]
Yeah you can refer this notebook https://github.com/lakshaychhabra/NSFW-Detection-DL/blob/master/NSFW.ipynb

**Comment by Venkatesh123w** [comment]
Hi, if  you  have, please provide train and test files here.

**Comment by lakshaychhabra** [comment]
Hi Please refer https://github.com/GantMan/nsfw_model and https://github.com/alex000kim/nsfw_data_scraper
